### ISSUE TITLE
NVDA announce as selected for non-selected "Setting" tab: A11y_AzureToolsStorageExplorer_Setting_ScreenReader

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/AzureStorageExplorer#8414]
### Preflight Checklist

- [x] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [x] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [x] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.36.2

### Architecture

x64

### Platform

Windows

### OS Version

Windows11

## GitHub Tags:

#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #A11ySev2; #BM_AzureToolsStorageExplorer_Win32_Dec2024; #Azure Tools Storage Explorer; #Screenreader; #WCAG1.3.1; #FTP; #NVDA; #Jaws; #STP; #Element:Tab;

## Environment Details:
Application: Microsoft azure storage explorer
Version: 1.36.2
OS: Windows 11 Enterprise 24H2
Build: 26100.2605
NVDA Version: 2024.4.1
JAWS Version: 2024.2312.53

## Repro Steps 

1. Turn on NVDA
2. Launch Azure Storage Explorer application.
3. TAB till "Setting" in left Navigation and press ENTER key.
4. Selected any other tab
5. TAB till setting tab 
6. Observe that NVDA announce as selected for non-selected "Setting" tab

## Actual Experience

NVDA announce as selected for non-selected "Setting" tab.
Note: Issue is repro with JAWS also

## Expected Experience

NVDA should not announce as selected for non-selected tab.

## User Impact
Screen reader user will get incorrect information as selected for non-selected tab

## Attachment
![Image](https://github.com/user-attachments/assets/c7d337d5-cd69-44bf-be6d-69b1f07e0e99)

JAWS
![Image](https://github.com/user-attachments/assets/b2f144c6-9c2f-4a94-9f35-e0028cf72bd5)

**Comment by InduPriya1805** [comment]
Rev:inse

**Comment by kapilvaishna** [comment]
Issue is repro in 1.37.0 version, please find repro attachment below.

![Image](https://github.com/user-attachments/assets/9853e560-de04-4a70-889c-56c4a198c450)


![Image](https://github.com/user-attachments/assets/0e0ddca8-949b-4d1f-ba1e-9315bc5e7fec)

**Comment by kupatkar99** [comment]
Verified the issue in below Environment:
Azure Storage Explorer version:1.38.0-dev(104)
Issue is fixed. Please find below added attachment for reference.
#Closed;

![Image](https://github.com/user-attachments/assets/b5fa9d1d-f2a1-4433-968a-e0b673bc5eab)
